styles/namemc/catppuccin.user.css

```diff
@@ -0,0 +1,582 @@
+      --bs-btn-focus-shadow-rgb: @blue;
+    }
+
+    #header {
+      form {
+        div.input-group.input-group-lg {
+          input.form-control {
+            background-color: @base;
+
+            &:focus {
+              border-color: @accent-color;
+              box-shadow: none;
+            }
+          }
+
+          button.btn.btn-primary {
+            background-color: @accent-color;
+            transition: all linear 0.15s;
+          }
+        }
+      }
+    }
+
+    .navbar-nav .nav-item a {
+      color: @text !important;
+
+      &:hover {
+        color: @subtext0 !important;
+      }
+      span {
+        color: @text !important;
+
+        &:hover {
+          color: @subtext0 !important;
+        }
+      }
+    }
+
+    .card {
+      background-color: @surface0 !important;
+      color: @subtext0 !important;
+      --bs-card-bg: @base !important;
+
+      .card-header {
+        border-color: @surface0 !important;
+      }
+    }
+
+    /* skin display link */
+    div.card-header {
+      span {
+        color: @text;
+
+        &:hover {
+          color: @accent-color !important;
+        }
+      }
+
+      strong a {
+        color: @accent-color !important;
+      }
+    }
+
+    /* skin display background */
+    .checkered {
+      background-image: none !important;
+      background-color: @mantle !important;
+    }
+
+    table.server-table {
+      > tbody > tr > td > a {
+        color: @accent-color !important;
+      }
+    }
+
+    /* skins page nav buttons */
+    .nav.nav-pills {
+      .nav-item .nav-link a {
+        color: @text !important;
+      }
+      .nav-link.active {
+        background-color: @accent-color !important;
+        color: @crust !important;
+      }
+    }
+
+    .page-item .page-link {
+      background-color: @surface0 !important;
+      border-color: @surface0 !important;
+      color: @text !important;
+
+      &.active {
+        background-color: @accent-color !important;
+      }
+
+      &:focus,
+      &:hover {
+        box-shadow: none !important;
+        color: @accent-color !important;
+      }
+    }
+
+    /* single skin page */
+    a.badge,
+    span.badge {
+      background-color: @accent-color !important;
+      color: @crust !important;
+    }
+
+    /* usernames page */
+    form#upcoming-form {
+      div.row {
+        label.col-form-label {
+          color: @text !important;
+        }
+
+        div.col {
+          .input-group {
+            .input-group-text {
+              background-color: @mantle !important;
+              border-color: @surface0 !important;
+            }
+          }
+        }
+      }
+
+      *.form-control,
+      *.form-select {
+        background-color: @mantle !important;
+        border-color: @surface0 !important;
+        box-shadow: none !important;
+        color: @subtext1 !important;
+
+        &:focus {
+          border-color: @accent-color !important;
+        }
```

Comment:
These shouldn't be scoped just for this page, the form-* elements are used all over.

Reply:
Also for the &:focus you need `box-shadow: 0 0 0 .25rem fade(@accent-color, 25%)`.